homeassistant/components/litterrobot/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "Litter-Robot",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/litterrobot",
-  "requirements": ["pylitterbot==2021.2.8"],
+  "requirements": ["pylitterbot==2021.3.1"],
```

Comment:
When bumping the library we want a link to a changelog or github commit compare range for the version bump in the PR description.